src/dom/__tests__/asyncHook.test.ts

```diff
@@ -1,5 +1,5 @@
 import { useState, useRef, useEffect } from 'react'
-import { renderHook } from '../../src/dom'
+import { renderHook } from '../index'
```

Comment:
I think `import { renderHook } from '..'` works too and might be a bit easier to maintain

Reply:
sure thing 👍🏼